scripts/templates/template.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Template
+parameters:
+- name: IMAGE_TAG
+  required: true
+- name: REPO_NAME
+  value: REPLACED_BY_SCRIPT
+  required: true
+metadata:
+  name: selectorsyncset-template
+objects:
+# replace old SSS w/ nothing so we don't have manual cleanup to do
+- apiVersion: hive.openshift.io/v1alpha1
+  kind: SelectorSyncSet
+  metadata:
+    name: osd-${REPO_NAME}
```

Comment:
@jewzaam Similar, `osd` prefix?

Reply:
@lisa review full context of the object, the `resources` property is an empty array.  This is so we have a SSS deployed with the current (and now old) name that has "osd-" prefix that wipes out the resources.  We can later delete it.  We don't have a way through the template to delete the SSS.  And if we do, I'd address it in a separate PR anyway as it'll apply to managed-cluster-config and all the exporters.